platform/admin/ui/src/main/webapp/templates/module/ModuleDetail.layout.handlebars

```diff
@@ -14,7 +14,7 @@
 <div class="page-layout full-height well">
     <div class="header clearfix">
         <div class="btn btn-success pull-right">
-            Export
+            Export System Configurations
```

Comment:
I added `System` to be verbose so there wouldn't be any confusion in the future. I got a bit paranoid that someone might wonder next time if `Export Configurations` exported the logging configuration, haha. Maybe that's a bit overkill though?

As for the `s` in `Configurations`, I was a bit conflicted. The button exports multiple `config` files, but technically as a whole they are DDF's `Configuration`. I'm still not too sure which is the best wording.